ru/mail/chrome/messenger/addressbook/abAddressBookNameDialog.dtd

```diff
@@ -36,4 +36,4 @@
  ***** END LICENSE BLOCK ***** -->
 
 <!-- Labels -->
-<!ENTITY name.label      "Имя адресной книги:">
+<!ENTITY name.label      "Имя адресной книги">
```

Comment:
I compared this with en-US file and probably it is not correct.
I have reverted this change. Should I commit the fixed file again?